### PR TITLE
Update sustainability element titles, removed deleted element, make targets unique

### LINKED ISSUE #419: Socioeconomic assessments of transition and decommissioning costs
# Background

This issue relates to the following CoST IDS elements proposed in the CoST IDS/OC4IDS review:

<details>

<summary> Decommission costs (Decommission) </summary>

## Decommission costs (Decommission)  

**Module:** Climate finance
**Indicator:** Climate management

### Disclosure format

> Disclose socio-economic assessments on transition and decommissioning costs (E.g.: [Doc]).

### OC4IDS mapping

> TBC

</details>

The original wording proposed by the consultant that carried out the research for the climate finance module was "Assessment of so-economic ‘transition costs’ of asset de-commissioning" with the following clarification:

> The underlying point was that in many climate finance projects there is a transition element, which means, of course, moving from one thing to another, and that, therefore, the ‘old thing’ needs to be taken into consideration when measuring the impact of the project. It may be operationally separate, but it’s part of the climate finance investment. 

## Discussion

There is already a 'Decommissioning cost forecast' element in the environment and climate module, which is mapped to `costMeasurements` (see https://github.com/open-contracting/infrastructure/issues/287), so my understanding is that this extra element is aimed at encouraging disclosure of a broader assessment of the of the socioeconomic costs associated with asset decommissioning in the context of transitioning to a low-carbon economy.

However, I'm struggling to find an example of such an assessment for an infrastructure project or an authoritative definition of "transition costs" that is applicable to this context.

@evelyndinora @mgraca-prado:

1. Do you have an example of such an assessment that you can share?
2. Can you provide a definition?

### PR DESCRIPTION
**Pull request by duncandewhurst**
**Related issues**

* Closes #419 
* Related to #415

**Description**

* Add a command to `manage.py` to update the CoST IDS sustainability elements from the [spreadsheet](https://docs.google.com/spreadsheets/d/165epI69oQ5YyL4-2q8VubFn9VuNham2Pi1u0P49id9o/edit#gid=0) in which they are maintained.
* Remove the "decommissioning costs" element per #419 
* Remove stages from element titles per https://github.com/open-contracting/infrastructure/issues/415#issuecomment-1797061932
* Include the module name in the targets generated by `update-sustainability-docs` so that they are unique

**Merge checklist**

- [x] [Log your changes](https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/contributing.html#logging-changes)

If there are changes to `project-schema.json` or `project-package-schema.json`:

- [ ] Update the examples:
  - [ ] `docs/examples/example.json`
  - [ ] `docs/examples/blank.json`
- [ ] Run `./manage.py pre-commit` to update `docs/_static/i8n.csv`

If you added a new definition to the schema, run `./manage.py pre-commit`.

If you added a new codelist:

  - [ ] Add an entry to `docs/reference/codelists.md`

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -808,6 +808,55 @@ def lint(filename, additional_properties):
     write_yaml_file(filename, elements)
 
 
+@cli.command()
+def update_sustainability_elements():
+    """Update mapping/sustainability.yaml"""
+
+    filename = basedir / 'mapping' / 'sustainability.yaml'
+
+    source = csv_reader(
```
Comment (jpmckinney) [inline manage.py:818]: Can you add a comment with a link to the spreadsheet itself?

File: manage.py [hunk 2]
```diff
@@ -808,6 +808,55 @@ def lint(filename, additional_properties):
     write_yaml_file(filename, elements)
 
 
+@cli.command()
+def update_sustainability_elements():
+    """Update mapping/sustainability.yaml"""
```
Comment (jpmckinney) [inline manage.py]: How does this interact with `update_sustainability_docs`?

**Review by jpmckinney — APPROVED** [review]
Some small comments. Feel free to merge once resolved.

File: manage.py (hunk 1 above)
Comment (duncandewhurst) [reply manage.py:818]: Done in https://github.com/open-contracting/infrastructure/pull/425/commits/6b1a950834b09d16e0cefcdf643b43ad164b5105

File: manage.py (hunk 2 above)
Comment (duncandewhurst) [reply manage.py]: This function updates the CoST IDS elements in `sustainability.yaml`, using the spreadsheet in which CoST maintain the sustainability elements as an input, whilst `update_sustainability_docs` updates the Markdown mapping documentation, using `sustainability.yaml` as an input.

I've updated the docstrings in https://github.com/open-contracting/infrastructure/pull/425/commits/6b1a950834b09d16e0cefcdf643b43ad164b5105

**Pull request merged**